src/reference/supported-platforms.md

```diff
@@ -4,52 +4,78 @@ short-title: Supported platforms
 description: The platforms that Flutter supports by platform version.
 ---
 
-## Support tiers
-
-We define three tiers of support for the platforms on
-which apps built with Flutter might be deployed:
-
-1. **Supported**<br>
-   Google-tested platforms that
-   are automatically tested on every commit
-   by continuous integration testing.
-1. **Best effort**<br>
-   Platforms that we intend to support through
-   coding practices,
-   but are only tested on an ad-hoc basis.
-1. **Unsupported**<br>
-   Platforms that we don't test or support.
-   
-{% comment %}
-**IMPORTANT NOTE**:
-When changing the minimum support version of macOS, Windows, or Linux,
-make sure to make a corresponding change in the get started pages:
-  * /src/get-started/install/macos.md
-  * /src/get-started/install/windows.md
-  * /src/get-started/install/linux.md
-{% endcomment -%}
-
-
-## Deploying Flutter
-
-As of the current stable release,
-support for deploying Flutter apps is shown in the
-following table:
+As of Flutter {{site.appnow.flutter}},
+Flutter supports the following hardware architectures and platform versions
+to run Flutter code.
+
+## Supported hardware architectures
+
+Flutter supports running apps on the following hardware architectures.
+
+{% assign chips = site.data.chipsets %}
+{% capture yes %}
+ <span class="material-symbols" 
+       style="color: #158477"
+       aria-label="The Flutter SDK supports the specified architecture on the specified target platform"
+       role="img">verified</span>
+{% endcapture %}
+{% capture no %}
+ <span class="material-symbols"
+       style="color: #D43324"
+       aria-label="The Flutter SDK does not support the specified
+       architecture on the specified target platform"
+       role="img">dangerous</span>
+{% endcapture %}
+{% capture beta %}
+ <span class="material-symbols"
+       style="color: #13C2AD"
+       aria-label="The Flutter SDK supports ARM64 in beta the only architecture on Windows"
+       role="img">gpp_maybe</span>
+{% endcapture %}
+{% capture na %}
+ <span class="material-symbols"
+       style="color: #DADCE0"
+       aria-label="No version of the Flutter SDK exists for the
+       specified architecture on the specified target platform"
+       role="img">do_not_disturb_on</span>
+{% endcapture %}
+
+| Platform |   x86   |   x64   |  ARM32  |   ARM64   | RV64GC   |
+|---|:---:|:---:|:---:|:---:|:---:|
+{%- for chip in chips %}
+  | {{chip.platform}} | {{chip.x86}} | {{chip.x64}} | {{chip.arm32}} | {{chip.arm64}} | {{chip.risc}} |
+{%- endfor %}
+{:.table.table-striped}
+
+<div aria-hidden="true" markdown="1">
+
+{{yes}} Supported in all channels.  
+{{no}} Unsupported in all channels.  
+{{beta}} Supported in Beta channel only.  
+{{na}} No version exists.  
+
+</div>
+
+## Supported operating systems
+
+Flutter offers three tiers of support for deploying apps to target platforms.
+
+* **Supported**: Google tests these platforms on every commit.
```

Comment:
Consider using a definition list here instead, since it seems to fit your use case and adds extra semantics.

Reply:
I tried that, but it looked like it added a bit too much page length. Let me check again.